test/test_transformers.py

```diff
@@ -87,37 +87,85 @@ def use_deterministic_algorithims(mode: bool, warn_only: bool):
 isSM5xDevice = torch.cuda.is_available() and torch.cuda.get_device_capability()[0] == 5
 isLessThanSM80Device = torch.cuda.is_available() and torch.cuda.get_device_capability()[0] < 8
 
-def get_rtol(true_value: torch.Tensor, computed_value: torch.Tensor) -> float:
-    deviation = true_value - computed_value
-    deviation = torch.abs(deviation / true_value)
-    # Fill in the nans with the default rtol
-    torch.nan_to_num_(deviation, nan=default_rtol[computed_value.dtype])
-    return deviation.max().item()
 
+def _check_equal(
+    golden: torch.Tensor,
+    reference: torch.Tensor,
+    test: torch.Tensor,
+    fudge_factor: float,
```

Comment:
nit: would be nice to have the meaning of the `fudge_factor` documented. I'm assuming it's a multiplicative factor applied to hp math vs. lp math errors. The output under test is allowed to be up to that factor * those errors away from the hp math output.